app/services/analytics_events.rb

```diff
@@ -1137,6 +1137,72 @@ def multi_factor_auth_enter_webauthn_visit(
     )
   end
 
+  # Max multi factor auth attempts met
+  def multi_factor_auth_max_attempts
+    track_event('Multi-Factor Authentication: max attempts reached')
+  end
+
+  # Multi factor selected from auth options list
+  # @param [Boolean] success
+  # @param [Hash] errors
+  # @param [String] selection
+  def multi_factor_auth_option_list(success:, errors:, selection:, **extra)
+    track_event(
+      'Multi-Factor Authentication: option list',
+      success: success,
+      errors: errors,
+      selection: selection,
+      **extra,
+    )
+  end
+
+  # Max multi factor auth attempts met
+  def multi_factor_auth_option_list_visit
+    track_event('Multi-Factor Authentication: option list visited')
+  end
+
+  # Multi factor auth phone setup
+  # @param [Boolean] success
+  # @param [Hash] errors
+  # @param [String] otp_delivery_preference
+  # @param [String] area_code
+  # @param [String] carrier
+  # @param [String] country_code
+  # @param [String] phone_type
+  # @param [Hash] types
+  # @param [Hash] pii_like_keypaths
+
+  def multi_factor_auth_phone_setup(success:,
```

Comment:
- remove blank line
- we can omit `pii_like_keypaths` in documentation since it doesn't make it to logs: https://github.com/18F/identity-idp/blob/main/lib/analytics_events_documenter.rb#L13-L15

```suggestion
  def multi_factor_auth_phone_setup(success:,
```

Reply:
taken care of.